README.md

```diff
@@ -1,11 +1,55 @@
-# Your GitHub Learning Lab Repository for Community Starter Kit
+# welcome bot: A Probot App
 
-Welcome to **your** repository for your GitHub Learning Lab course. This repository will be used during the different activities that I will be guiding you through to learn about making your project friendlier for collaboration.
+DESCRIPTION PLACEHOLDER
```

Comment:
- this project helps to understand how github flow works

- bot is really helpful with first steps at github